app/controllers/application_controller.rb

```diff
@@ -1,4 +1,14 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password) }
+  end
+
   def current_user
     @user = User.first
   end
```

Comment:
- [ ] In this milestone, we are required to use the [Devise](https://github.com/heartcombo/devise) gem for the authentication. 🤓🛠

- [ ] Please note that the `current_user` method in `ApplicationController` should be removed. Devise provides a method to do this. 👍

- [ ] You can check the Devise [getting started link](https://github.com/heartcombo/devise).